ompi/mca/osc/monitoring/configure.m4

```diff
@@ -1,22 +1,98 @@
-# -*- shell-script -*-
-#
-# Copyright (c) 2016 Inria.  All rights reserved.
-# $COPYRIGHT$
+dnl -*- shell-script -*-
+dnl
+dnl Copyright (c) 2016-2017 Inria.  All rights reserved.
+dnl $COPYRIGHT$
+dnl
+dnl Additional copyrights may follow
+dnl
+dnl $HEADER$
+dnl
+
+# mca_ompi_osc_monitoring_generate_templates
 #
-# Additional copyrights may follow
+# Append to file $1 an array of components by listing component names in $2.
```

Comment:
Just to clarify this comment: this function does not *append* to the end of an existing file -- it *overwrites* file $1.